crates/fuel-core/src/schema/tx/types.rs

```diff
@@ -396,16 +400,27 @@ impl Transaction {
         ctx: &Context<'_>,
     ) -> async_graphql::Result<Option<TransactionStatus>> {
         let id = self.0.id();
-        let query = TransactionQueryContext(ctx.data_unchecked());
+        let query: &Database = ctx.data_unchecked();
         let txpool = ctx.data_unchecked::<TxPool>();
-        get_tx_status(id, &query, txpool).await.map_err(Into::into)
+        match query
```

Comment:
Why did you extract this functionality form the `get_tx_status` method?

Reply:
There was a brief lifetime error where it wouldn't pass in that I fixed but forgot to change this back